### ISSUE TITLE
Photos and Blogger lightbox 

### ISSUE THREAD

**Issue by Zeinobia** [OpenLiveWriter/OpenLiveWriter#185]
Currently I am using OWL on Windows 10 to post to a Blogger Blog. Now I noticed  that the Blogger Built-in lightbox for photos does not work with the photos uploaded with OWL. 
I do not know if you can add the caption option to Blogger's option in OWL , it will be a great thing to save time.
Thanks

**Comment by Dinchy87** [comment]
Yep same here, prior to this if i use picture uploads from windows live writer it was also the same thing it uploads the picture but the lightbox option just opens the pictures uploaded manually from blogger.com. But the blogger.com option for picture upload adds some style extensions to it like if we align the picture left then center is ats a margin left and right that is not ok. This is why i like the picture upload from within live writer it doesnt add anything bad.

**Comment by Dinchy87** [comment]
So i have checked this a little bit. The pictures posted from live writer are on a url named *googleusercontent.com/.... and so on and this pictures does not open in the lightbox blogger function for pictures. Instead it opend the picture in full res in a new tab/window. 

What i wanted to ask now is can we manipulate it somehow that live writer publishes the pictures to the same directory like Blogger.com does when publishing it?! the url for blogger pictures is *blogspot.com/...

i think this makes the difference how it opens the Picture in lightbox or not. I wish we would have the ability to choose where to post the Picture Blogger or the poster own Google picture storage (googleusercontent).

**Comment by xbmcnut** [comment]
Same here. Would love the lightbox feature to still work when uploading images using OLW.

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
